src/Symfony/Component/Mailer/Bridge/AhaSend/composer.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "symfony/ahasend-mailer",
+    "type": "symfony-mailer-bridge",
+    "description": "Symfony AhaSend Mailer Bridge",
+    "keywords": [],
```

Comment:
please use the keywords like in the other mailer bridges

Reply:
Keywords are empty for other mailer bridges as well.